x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/types.ts

```diff
@@ -20,7 +20,6 @@ export interface IFlashMessage {
 // @see EuiGlobalToastListToast for more props
 export interface ToastOptions {
   iconType?: string;
-  id?: string;
-  text?: ReactChild; // Additional text below the message/title, same as IFlashMessage['description']
-  toastLifeTimeMs?: number; // Allows customing per-toast timeout
+  text?: string; // Additional text below the message/title, same as EuiToast['text']
```

Comment:
The Notification service text is `string | MountPoint` but luckily it looked like all our usages of this option were `string` so I omitted `MountPoint` for now as it would be harder to deal with.